### PR TITLE
Task4: completed task 4 'functions'

### PR DESCRIPTION
**Pull request by IrynaKazei**
Please review my  task4

### CODE REVIEW

File: tasks/functions/functions.js [hunk 1]
```diff
@@ -15,31 +15,48 @@ function add(a, b) {
...
 
+
 /**
  * write fuction that checks is number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+    let x = n % 2;
+    if (x == 0) {
```
Comment (dshaplyko) [inline tasks/functions/functions.js:28]: you can put the statement here:
if ( n % 2 == 0) {...}

File: tasks/functions/functions.js [hunk 2]
```diff
@@ -15,31 +15,48 @@ function add(a, b) {
...
 }
 
 /**
  * write function that returns word google with given numbers of "o" symbols
  * e.g getGoogle(5) should return "gooooogle"
  */
 function getGoogle(n) {
-
+    let google = "g";
+    for (let j = 1; j <= n; j++) {
```
Comment (dshaplyko) [inline tasks/functions/functions.js:56]: it should have been done using repeat() function :)

return 'g' + 'o'.repeat(n) + 'gle'

File: tasks/functions/functions.js [hunk 3]
```diff
@@ -52,7 +69,12 @@ function getGoogle(n) {
  * }
  */
 function getUser(firstName, lastName, age) {
```
Comment (dshaplyko) [inline tasks/functions/functions.js:71]: please try to use default function parameters here, such as 
function getUser(firstname = null, ...) {
return {
firstname,
...
}
}

File: tasks/functions/functions.js (hunk 1 above)
Comment (IrynaKazei) [reply tasks/functions/functions.js]: thanks, fixed

File: tasks/functions/functions.js (hunk 2 above)
Comment (IrynaKazei) [reply tasks/functions/functions.js]: thanks, fixed

File: tasks/functions/functions.js (hunk 3 above)
Comment (IrynaKazei) [reply tasks/functions/functions.js]: thank you very much. Fixed.)

**Pull request merged**